docs/docs/language_reference/schema/index.md

```diff
@@ -344,14 +358,15 @@ type Post implements Webpage {
 ## Entrypoints
 
 An entrypoint gives us an initial set of vertices to work with. Each Trustfall schema needs
-a root element called `schema` with a field `query` that is our entrypoint type - the starting
+a root element called `schema` with a field `query` that is our entrypoint type — the starting
 point of the query.
 
 In the entrypoint we want to provide the means to query our data source for the initial set of
 vertices we'll consider and filter or transform. For data sources like APIs we'll usually define
-some queries that map to API queries. For data types where all the data is already present such
-as a schema representing something like a json file our entrypoint type may well have no queries
-and just properties and/or edges. 
+some queries that map to API queries. If all of our data is already available, we might have no
+queries and just properties and edges. Although, even in that case we may want to add some
+queries to simplify some operations that would be complicated or impossible to express in a
+Trustfall query.
```

Comment:
I think this used "queries" where I would have used "entrypoints" per the term we just defined.

Suggested tweak:
```suggestion
some entrypoints that map to API endpoints. If all of our data is available without restrictions,
like in a SQL database, we might just define simple entrypoints named after
(and returning a list of) each vertex type. Although, even in that case we may want to
add some more sophisticated entrypoints, to simplify some operations that might
be complicated or impossible to express in a Trustfall query.
```